src/main/java/duke/Task.java

```diff
@@ -0,0 +1,39 @@
+package duke;
+
+public abstract class Task {
+
+    private String description;
+    private boolean isDone;
+
+    public Task(String desc) {
+        this.description = desc;
+        this.isDone = false;
+    }
+
+    public String getStatus() {
+        return isDone ? "V" : "X";
+    }
+
+    public void setStatus(boolean status) {
+        this.isDone = status;
+    }
+
+    public String getDescription() {
+        return this.description;
+    }
+
+    public int getStatusAsInt() {
+        return isDone ? 1 : 0;
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
```

Comment:
This might just be me but I feel that some of these methods are slightly redundant